cluster/gce/gci/configure-helper.sh

```diff
@@ -1265,11 +1265,18 @@ EOF
   fi
 }
 
+# Create kubeconfig files for control plane components.
 function create-kubeconfig {
   local component=$1
   local token=$2
   echo "Creating kubeconfig file for component ${component}"
   mkdir -p "/etc/srv/kubernetes/${component}"
+
+  local kube_apiserver="localhost"
+  if [[ ${KUBECONFIG_USE_HOST_IP:-} == "true" ]] ; then
+    kube_apiserver=$(hostname -i)
```

Comment:
This is not ok, it produces a kubeconfig that sends a replayable credential over an unverified TLS connection. Reverting in 1.20.1, can bring this back in in 1.21 with a verified TLS connection if desired.